### ISSUE TITLE
MKS tft32 M600 support

### ISSUE THREAD

**Issue by docean24** [makerbase-mks/MKS-TFT#353]
Can support please be added for M600 commands? Currently M600 will pause a print but the touch screen does not recognize that the print has paused and there is no way to resume the print. This is for advanced filament out functionality so assigning a "more" button will not help.

**Comment by makerbase-mks** [comment]
This feature, we can implement in the next release. 
This logic is M600-->display pause and machine pause---->click on the touch screen to resume printing. Do you need such a function?

**Comment by marcohaddad** [comment]
Hello,
That's great! I'll love to have it too!
Best regards,

**Comment by docean24** [comment]
Yes that's what I need thank you. When will the next release be?

**Comment by Stady234** [comment]
I have been waiting for almost a year for this to be added... more specifically... if ANY printer pause/user must click old screen button to continue - option to be added.. but still waiting...

**Comment by makerbase-mks** [comment]
We are preparing for open source work in 2020. I think if we have time, we will definitely do this.

**Comment by manukieli** [comment]
Very good new.
M600 command will helpfull.
Can add it to TFT28??

**Comment by rost76** [comment]
I am also waiting for tft32

**Comment by Claymore61** [comment]
I would like this function too.
I was thinking I just hadn’t coded it correctly 😂😂

**Comment by Claymore61** [comment]
Out of interest, when is the next release likely to be ?
I’m very new to this!

**Comment by bartofborg** [comment]
this is needed! any time line revision yet?

**Comment by tanaes** [comment]
Adding my interest for this feature as well!

**Comment by Telekmac** [comment]
Please add this. You should add support for all functions that require user input sent from Marlin. Especially m600

**Comment by betocabus** [comment]
M600 is the only funtion I want to change from traditional LCD to touch LCD, all of them doesnt work! I use M600 every day!
